src/modules/group/group.handler.ts

```diff
@@ -0,0 +1,81 @@
+import type { AppRouteHandler } from "@/common/lib/types";
+import type { TSelectGroupSchema } from "@/db/schemas/group.model";
+
+import * as HTTPStatusCodes from "@/common/utils/http-status-codes.util";
+
+import type { TCreateGroupRoute, TDeleteGroupRoute } from "./group.routes";
+
+import { createGroupRepository, deleteGroupRepository } from "./group.repository";
+
+export const createGroup: AppRouteHandler<TCreateGroupRoute> = async (c) => {
+  const user = c.get("user");
+  const payload = c.req.valid("json");
+
+  if (!user) {
```

Comment:
I missed this initially but when ever we are returning a response it is better to log the response using the logger, for reference check create category handler and log these events so we can look at them later.
Also, we have a activity model, we need to log these activities in that table.